mne/io/constants.py

```diff
@@ -136,10 +136,11 @@ def __setattr__(self, attr, val):
 FIFF.FIFF_DESCRIPTION    = FIFF.FIFF_COMMENT # (Textual) Description of an object
 FIFF.FIFF_DIG_STRING     = 234          # String of digitized points
 FIFF.FIFF_LINE_FREQ      = 235    # Line frequency
-FIFF.FIFF_CUSTOM_REF     = 236    # Whether a custom reference was applied to the data
+FIFF.FIFF_CUSTOM_REF     = 236    # Whether a custom reference was applied to the data (whoops, overlaps...)
```

Comment:
Hi guys,
We should do some concerted work here with Elekta to keep the definitions in line with the FIFF format documentation. The current documentation (as well as neuromag libraries and source codes) can be found in the fiff_support package at http://www.aston.ac.uk/lhs/research/centres-facilities/brain-centre/facilities-clinical-services/meg-studies/downloads/. We are already defining some new extensions...